pkg/synopsysctl/cmd_update.go

```diff
@@ -1394,13 +1396,58 @@ var updatePolarisReportingCmd = &cobra.Command{
 	},
 }
 
+// updateBDBACmd updates a BDBA instance
+var updateBDBACmd = &cobra.Command{
+	Use:           "bdba",
+	Example:       "",
+	Short:         "Update a BDBA instance",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+		// Check the Number of Arguments
+		if len(args) != 0 {
+			cmd.Help()
+			return fmt.Errorf("this command takes 0 argument, but got %+v", args)
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// Get the flags to set Helm values
```

Comment:
add Get method to get the previous stored value and merge those values with new values